biceps-model/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+
+plugins {
+    id("com.draeger.medical.java-conventions")
+    id("org.somda.sdc.xjc")
+}
+
+val jaxb: Configuration by configurations.creating
+val schemaDir = "src/main"
+
+
+dependencies {
+    api(libs.org.glassfish.jaxb.jaxb.core)
+    api(libs.org.glassfish.jaxb.jaxb.runtime)
+    api(libs.jakarta.xml.bind.jakarta.xml.bind.api)
+    api(libs.io.github.threeten.jaxb.threeten.jaxb.core)
+    api(libs.org.jvnet.jaxb.jaxb.plugins)
+    api(libs.org.checkerframework.checker.qual)
+    api(libs.org.jetbrains.annotations)
+
+    testImplementation(libs.org.junit.jupiter.junit.jupiter.api)
+    testImplementation(libs.org.junit.jupiter.junit.jupiter.engine)
+
+    jaxb(libs.org.jetbrains.annotations)
+    jaxb(libs.org.jvnet.jaxb.jaxb.plugin.annotate)
```

Comment:
In the same vein as the other remark in this file, we don't use the annotate plugin